apps/sms/views/conversation/js/conversation.js

```diff
@@ -14,7 +14,8 @@
          EventDispatcher,
          SelectionHandler,
          TaskRunner,
-         MessagingClient
+         MessagingClient,
+         MozMobileConnsClient
```

Comment:
__optional:__ not sure, but it seems longer name (```MozMobileConnectionsClient```) will be better for the sake of clarity like you use for the shim name.